sdk/storage/Azure.Storage.Blobs/src/BlobClient.cs

```diff
@@ -1020,6 +1028,26 @@ public BlobClient(Uri blobUri, TokenCredential credential, BlobClientOptions opt
             bool async = true,
             CancellationToken cancellationToken = default)
         {
+            // TODO uncomment when upload from file gets its own implementation
```

Comment:
to be addressed in this PR?

Reply:
No. This is to do with future work to give upload from file its own implementation separate from upload from stream. However, such an implementation will be incompatible with this feature, and so it will need to revert to upload from stream. I can clarify this.